coriolisclient/v1/logging.py

```diff
@@ -0,0 +1,182 @@
+# Copyright (c) 2019 Cloudbase Solutions Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import datetime
+import json
+import requests
+import websockets
+from six.moves.urllib import parse as urlparse
+from keystoneauth1.exceptions.catalog import EndpointNotFound
```

Comment:
NITS:
* `websockets`, `six` and `keystoneauth` should be in a separate block for 3rd party packages
* `six` would come after `keystoneauth`